components/organisms/ProfileForm/ProfileFormContent.js

```diff
@@ -83,9 +108,16 @@ const ProfileFormContent = ({ profile: { email, firstName, lastName }, onSubmit,
     submit: onSubmit,
   };
 
+  const isAvatarUrl = temporaryUrl || avatarUrl;
+
   return (
     <FormWrapper>
       <StyledTitle>Profile</StyledTitle>
+      {isAvatarUrl && (
+        <AvatarWrapper>
+          <AvatarImg src={isAvatarUrl} />
```

Comment:
nit: `isAvatarUrl` name means that this is boolean, but we use it for src. I think need more clear name